### PR TITLE
SEO review

### PR DESCRIPTION
**Pull request by Williangalvani**
This adds some links back to bluerobotics.com, fixes old/broken links, and updates old information along the way.

### CODE REVIEW

File: getting-started/installation.md [hunk 1]
```diff
@@ -2,11 +2,8 @@
 
 ## QGroundControl
 
-Download and install QGroundControl using one of the links below.
+Download and install QGroundControl from the [QGroundControl website](https://docs.qgroundcontrol.com/en/getting_started/download_and_install.html).
```
Comment (jaxxzer) [inline getting-started/installation.md]: revert this, we still host the 'recommended' version in our bucket.

File: introduction/applications.md [hunk 2]
```diff
@@ -14,7 +14,7 @@ ArduSub provides the functionality needed for a wide variety of applications fro
 
 ## ArduSub in Use
 
-These videos show ArduSub in use on the Blue Robotics BlueROV1 and BlueROV2. Keep in mind that *ArduSub* can be used on many different ROV designs from unique DIY configurations to professional vehicles.
+These videos show ArduSub in use on the Blue Robotics [BlueROV1](https://bluerobotics.com/store/retired/bluerov-r1/) and [BlueROV2](https://bluerobotics.com/store/rov/bluerov2/). Keep in mind that *ArduSub* can be used on many different ROV designs from unique DIY configurations to professional vehicles.
```
Comment (jaxxzer) [inline introduction/applications.md]: Maybe we should update our video examples @rjehangir

File: introduction/features.md [hunk 3]
```diff
@@ -69,13 +69,15 @@ Please let us know if you have tested ArduSub on another platform!
 
 In addition to the standard onboard sensors (IMU, compass), the ArduSub controller directly supports a number of external sensors including:
 
-- Pressure/depth sensors for measurement and auto depth-hold ([example](https://www.bluerobotics.com/store/electronics/bar30-sensor-r1/))
-- GPS for position at the surface (does not work underwater)
+- Pressure/depth sensors for measurement and auto depth-hold ([such as the Bar30](https://www.bluerobotics.com/store/electronics/bar30-sensor-r1/))
+- GPS for position at the surface
```
Comment (jaxxzer) [inline introduction/features.md]:
```suggestion
- Underwater positioning systems like USBLs and the [waterlinked positioning system](linnk)
```

Comment (jaxxzer) [reply introduction/features.md]: just omit this line

File: introduction/applications.md (hunk 2 above)
Comment (Williangalvani) [reply introduction/applications.md]: It would be cool if we had videos for most of the applications listed there.

**Pull request merged**